### PR TITLE
CU-2f93hm - Retina Logo not set

### PR DESCRIPTION
**Pull request by rohitp-bsf**
### Description
fix: Default for ratina and other options was not working

### Screenshots
- NA

### Types of changes
- Added a default as an updated value.

### How has this been tested?
- Set the new value to the option, reload the customizer and check if updated values appear.

### Checklist:
- [ ] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Comment by Nikschavan** [comment]
Task linked: [CU-2f93hm Retina Logo not set](https://app.clickup.com/t/2f93hm)

**Comment by Nikschavan** [comment]
Task linked: [CU-2f82qd Fix: LearnDash (3.0 template) - Overwrite LearnDash color option is not being saved in master](https://app.clickup.com/t/2f82qd)

File: inc/compatibility/lifterlms/customizer/sections/layout/class-astra-lifter-general-configs.php [hunk 1]
```diff
@@ -36,15 +36,18 @@ public function register_configuration( $configurations, $wp_customize ) {
 				 * Option: Course Columns
 				 */
 				array(
-					'name'              => ASTRA_THEME_SETTINGS . '[llms-course-grid]',
```
Comment (vrundakansara) [inline inc/compatibility/lifterlms/customizer/sections/layout/class-astra-lifter-general-configs.php:39]: Why have we removed the name?

**Review by vrundakansara — CHANGES_REQUESTED** [review]

File: inc/compatibility/lifterlms/customizer/sections/layout/class-astra-lifter-general-configs.php (hunk 1 above)
Comment (rohitp-bsf) [reply inc/compatibility/lifterlms/customizer/sections/layout/class-astra-lifter-general-configs.php]: It was mistakenly removed @vrundakansara! I have just reverted this change!

**Review by vrundakansara — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**